crates/client/src/pages/login.rs

```diff
@@ -1,5 +1,4 @@
 use crate::helpers::get_by_id::get_by_id;
```

Comment:
About move, client really have something else than pages (or proper name for that is a components)?

Reply:
I can structure it as /pages/account /pages/error etc.

Reply:
But I don't think this is necessary.

Reply:
I meant, why we need this `pages/` directory and module? What isn't a component/page?

Reply:
I prefer routes(pages) instead of components.

Reply:
[SPA](https://en.wikipedia.org/wiki/Single-page_application) and reactive UI doesn't work in that way, whole application should be more understood like one thing, not many different and unrelated pages. E.g. you have channel list where near to you place a chat content, if chat content will change for e.g. voice channel we didn't reload page, we only refresh this one container. This approach is constructed from many dynamic components, and exists is one "process".